### ISSUE TITLE
Any plans for building for linux arm64?

### ISSUE THREAD

**Issue by masip85** [pyutils/line_profiler#44]
This package is not available in pip or conda for linux arm64. Are there any plans for these architectures?

**Comment by Erotemic** [comment]
I would accept a PR that added support for this.

I'm currently working on migrating to CircleCI in #43 so any updates to the build and publish process on the CI should target that branch.

**Comment by thedimlebowski** [comment]
I managed to install line_profiler on aarch64 by installing manually `scikit-build` and `ninja`.
```
conda install -c conda-forge ninja
conda install -c conda-forge scikit-build
git clone https://github.com/pyutils/line_profiler.git
cd line_profiler && python setup.py install
```

**Comment by Erotemic** [comment]
Fixed by #76
